### DISCUSSION TITLE
dsfo.exe flagged as Trojan:Win32/Mamson.A!ac in antivirus

### DISCUSSION THREAD

**Discussion by patrikroy** [RedGreenBlue09/WFAv7_Installer#5] [General]
While cloning `WFAv7_Installer` using Github Desktop, Windows Defender flagged `dsfo.exe` as a trojan and put it in quarantine.
Any thoughts why?

**Comment by RedGreenBlue09** [comment]
That's a false positive. Try excluding it. The program is used to extract a part of a binary file. I think it's marked because it do operations like what malwares do to overwrite boot sector, but why "trojan" then IDK.
